src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -224,14 +225,12 @@ async function validatePythonAlias(pyAlias: PythonAlias): Promise<string | undef
     }
 }
 
-function convertToVenvCommand(command: string, platform: NodeJS.Platform, separator?: string): string {
+function getVenvActivateCommand(platform: NodeJS.Platform): string {
```

Comment:
I kind of liked having the `convertToVenvCommand`.  I'm not sure it's immediately obvious why you're using `joinCommands` and then `getVenvActivateCommand.